common/changes/@itwin/frontend-tiles/andremig-mxprovider_2024-05-30-14-29.json

```
@@ -0,0 +1,10 @@
+{
+  "changes": [
+    {
+      "packageName": "@itwin/frontend-tiles",
+      "comment": "Refactor FrontendTiles",
```

```suggestion
      "comment": "",
```

Changelogs record changes of interest to users. No user cares that we moved some code around internally. (Also, committing this change will trigger the various build pipelines that you would otherwise need to manually, individually trigger)).